chain/client/src/client.rs

```diff
@@ -1425,6 +1425,8 @@ impl Client {
         let head = self.chain.head()?;
         if let Some(next_epoch_id) = self.get_next_epoch_id_if_at_boundary(&head)? {
             self.forward_tx(&next_epoch_id, tx)?;
+        } else {
```

Comment:
> looking at the place where this is called in process_tx_internal it appears that if an active validator received a tx directly, the tx will never be forwarded before this change, resulting in very long processing times

Yes that is exactly the intention. One python test failed because of this.

> But if this is indeed your intent here, the method needs to be renamed?

What do you suggest we rename it to?